development-defaults.ini

```diff
@@ -173,7 +173,6 @@ group  = "Taking an unassigned Group badge (group leader must be present)"
 [[event_location]]
 
 [[dept_head_overrides]]
-staff_support = "Jack Boyd"
```

Comment:
Unrelated fix for Stops.  This doesn't really belong in the checked-into-source-control ``development-defaults.ini`` anyway.